Meilensteine/Meilenstein_04/frontend/src/inputValidators.js

```diff
@@ -76,3 +76,16 @@ export function arrayEveryEntry(validator) {
     return _.join(errors);
   };
 }
+
+// neu: Luke
+export function validateEmail(email) {
+  // Regulärer Ausdruck zur Überprüfung der E-Mail-Adresse
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  
+  // Überprüfung, ob die E-Mail-Adresse dem regulären Ausdruck entspricht
+  if (emailRegex.test(email)) {
+      return null; // Die E-Mail-Adresse ist gültig
+  } else {
+      return 'Ungültige E-Mail-Adresse'; // Die E-Mail-Adresse ist ungültig
+  }
```

Comment:
```suggestion
  }
  return 'Ungültige E-Mail-Adresse'; // Die E-Mail-Adresse ist ungültig
```
Nur ne Kleinigkeit :)